packages/i18n/src/test/create-i18n.js

```diff
@@ -191,4 +191,63 @@ describe( 'createI18n', () => {
 	} );
 } );
 
+describe( 'i18n filters', () => {
```

Comment:
There's one more thing that I thing is worth testing: interaction of the default `i18n` with the default `hooks`:
```js
import { __ } from '@wordpress/i18n';
import { addFilter } from '@wordpress/hooks';

test( () => {
  addFilter( 'i18n.gettext', ... );
  expect( __( 'hello' ) ).toBe( ... );
} );
```

Reply:
Added packages/i18n/src/test/default-i18n.js. Thanks!